### PR TITLE
Localization: Refactor and clean up transforms

### PR DESCRIPTION
**Pull request by Flova**

### CODE REVIEW

File: bitbots_localization/include/bitbots_localization/localization.h [hunk 1]
```diff
@@ -219,8 +219,8 @@ class Localization {
 
   void getMotion();
 
-  geometry_msgs::Vector3 movement_;
-  geometry_msgs::Vector3 movement2_;
+  geometry_msgs::Vector3 cartesian_movement_;
```
Comment (NFiedler) [inline bitbots_localization/include/bitbots_localization/localization.h]: As in #43, I woult prefer linear instead of cartesian

Comment (Flova) [reply bitbots_localization/include/bitbots_localization/localization.h]: I know. #43 gets modified first. I just wanted to build this one on top of it and wait (draft) until the other one is finished.

Comment (NFiedler) [reply bitbots_localization/include/bitbots_localization/localization.h]: alright

Comment (Flova) [reply bitbots_localization/include/bitbots_localization/localization.h]: Done

**Comment by Flova** [comment]
> LGTM. Can the evaluation stuff (frames and message) be turned off easily? I think we should be able to turn it off as it is useless most of the time. But if we need to implement that it would be another issue/pr.

Okay I create an issue for that.

**Pull request merged**